app/gui/src/dashboard/layouts/Drive/Categories/categoriesHooks.tsx

```diff
@@ -0,0 +1,411 @@
+/**
+ * @file
+ *
+ * Hooks for working with categories.
+ * Categories are shortcuts to specific directories in the Cloud, e.g. team spaces, recent and trash
+ * It's not the same as the categories like LocalBackend
+ * TODO: Improve performance and add ability to subscribe to individual category values
```

Comment:
ideally this TODO should be linked to an issue too

Reply:
I think the todo here isn't strictly necessary as the perf impact is not measured yet.